app/assess/models/assessment_flow.py

```diff
@@ -0,0 +1,62 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class AssessmentFlow:
+    sections: list
+    section_name: list
+    questions: list
+    question: list
+    fields: list
+
+    @staticmethod
+    def as_json(assessment_json, page_index):
+        return AssessmentFlow(
+            sections=AssessmentFlow.get_sections(assessment_json),
+            section_name=AssessmentFlow.get_section_name(
+                assessment_json, page_index
+            ),
+            questions=AssessmentFlow.get_questions(
+                assessment_json, page_index
+            ),
+            question=AssessmentFlow.get_question(assessment_json, page_index),
+            fields=AssessmentFlow.get_fields(assessment_json, page_index),
```

Comment:
⛏️ Maybe these could be moved into variables in this function for better readability?